docs/_sidebar.md

```diff
@@ -1,26 +1,26 @@
-### [Forge Wiki Home](Home.md)
+# [Forge Wiki Home](Home.md)
```

Comment:
Appreciate the wiki touches
I also thought it weird to start with the third heading but when it's in the Wiki it looked way too huge imo

Reply:
Again, it breaks the title hierarchy. Usually I'd advocate for not using wrong tags for display reasons, but it's a minor issue  and I can revert it.

Reply:
for this I'd insist since it takes way too much space otherwise, especially if you browse on mobile
if there's a better way to increase the text size slightly feel free to switch to that though
thanks!